MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,191 @@
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="LostFocus">
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.LostFocus;
+{% endhighlight %}
+{% endtabs %}
+
+#### PropertyChanged
+
+If the commit mode is `PropertyChanged`, the value will be committed immediately when it is changed.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="PropertyChanged">
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.PropertyChanged;
+{% endhighlight %}
+{% endtabs %}
+
+#### Manual
+
+If the commit mode is `Manual`, the value should be committed manually by calling the [SfDataForm.Commit] method.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+        xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="Manual">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.Manual;
+{% endhighlight %}
+{% endtabs %}
+
+The following code commits the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight C# %}
+this.dataForm.Commit();
+{% endhighlight %}
+{% endtabs %}
+
+N> On manual commit, manual validation will be called to validate the properties before the commit.
+
+## Value converter attribute
+
+To show the original value in different format or as different value, use the [DataFormValueConverter] attribute.
+
+#### Changing original value of the DataForm property value using converter
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormValueConverter(typeof(StringToDateTimeConverter))]
+    public string DateTime { get; set; }
+
+public class StringToDateTimeConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        if (value == null || string.IsNullOrEmpty(value.ToString()))
+        {
+            return DateTime.Now;
+        }
+        else
+        {
+            DateTime dateTime;
+            DateTime.TryParse((string)value, out dateTime);
+            return dateTime;
+        }
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return value.ToString();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the editor will display the original value as `DateTime`. While committing, it is converted to `string` and stored in data object.
+
+## Read only mode
+
+`DataForm` can be disabled by setting the [IsReadOnly] property of the data form.
```

Comment:
The `DataForm` can be disabled by setting the [IsReadOnly] property of the data form.

Reply:
content changed